quickwit/quickwit-control-plane/src/ingest/ingest_controller.rs

```diff
@@ -591,17 +592,17 @@ mod tests {
             GetOrCreateOpenShardsSubrequest {
                 index_id: "test-index-0".to_string(),
                 source_id: source_id.to_string(),
-                closed_shards: Vec::new(),
             },
             GetOrCreateOpenShardsSubrequest {
                 index_id: "test-index-1".to_string(),
                 source_id: source_id.to_string(),
-                closed_shards: Vec::new(),
             },
         ];
+        let closed_shards = Vec::new();
         let unavailable_ingesters = vec!["test-ingester-0".to_string()];
         let request = GetOrCreateOpenShardsRequest {
             subrequests,
+            closed_shards,
```

Comment:
just out of curiosity, why this is not inlined? convention?

Reply:
No reasons but it does not matter.